.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,22 @@
-**I'm submitting a ...**
-- [ ] bug report
-- [ ] feature request
+# Template to submit an issue
+
+<!-- Thank you for contributing by opening an issue! Please review this guide before submitting your issue.
+    - If you are opening an issue because you would like to propose a new feature, write the title as "Feature Request:" followed by a short description of the feature.
+    - Make sure that you are using the correct version of Node.js. You need to have version v6+ for everything to work fine.
+    - Ensure that your new issue conforms to the contribution guidelines: https://github.com/UdacityMobileWebScholarship/showcase-app/blob/master/.github/CONTRIBUTING.md
+    - Please do read guidelines :
+    https://github.com/UdacityMobileWebScholarship/showcase-app/README.md
+    - TODO - After reading this template you can remove this template while submitting an issue -->
+
+
+#### I'm submitting this issue because:
```

Comment:
@optimistanoop Please keep it as it was before. No need to add extra checkboxes as added things can be included in **Bug Report**.